app/src/common/error.ts

```diff
@@ -1,30 +1,30 @@
 /* eslint-disable max-classes-per-file */
 export class JenkinsAppError extends Error {
-  constructor(message: string) {
-    super(message);
-  }
+	constructor(message: string) {
```

Comment:
What was the decision we came up here again? 8 spaces? 2 tabs?

Reply:
Tabs and the `.editorconfig` will let you set it to however many spaces you want those tabs to be :) GitHub sets the tabs to be 4 spaces wide

https://hello.atlassian.net/wiki/spaces/PF/pages/1424951560/Arc+Coding+Conventions